BLSQ/iaso
# WC2-176 : Change frontend, backend, update tests

In workflow changes, the order of the field mappings was reversed between web and mobile.
This reverses it in web frontend and backend.

Related JIRA tickets : WC2-176

## Self proofreading checklist


## Changes

Reversed the order of the mapping Source Field > Target Field in frontend
Reversed the order of the mapping Source Field > Target Field in backend
Fixed the API tests to take it into account

## How to test

Go to Beneficiary type > Workflow and add or edit changes in a workflow
Verify that the order is correct.

(See video below)

## Print screen / video


https://user-images.githubusercontent.com/383344/223390845-977b6910-7dee-4e39-8fe8-626a8070797f.mov
